submarine-core/pom.xml

```diff
@@ -124,6 +124,10 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Why do you need to import `jackson` jar?

Reply:
The error that jackson class is not found if jackson-core is not imported